src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicWithHorizon.hpp

```diff
@@ -67,11 +67,13 @@ struct EvolutionMetavars<3, InitialData, BoundaryConditions>
 
   struct AhA {
     using temporal_id = ::Tags::Time;
-    using tags_to_observe =
-        tmpl::list<StrahlkorperGr::Tags::AreaCompute<frame>,
-                   StrahlkorperGr::Tags::IrreducibleMassCompute<frame>,
-                   StrahlkorperTags::MaxRicciScalarCompute,
-                   StrahlkorperTags::MinRicciScalarCompute>;
+    using tags_to_observe = tmpl::list<
+        StrahlkorperGr::Tags::AreaCompute<frame>,
+        StrahlkorperGr::Tags::IrreducibleMassCompute<frame>,
+        StrahlkorperTags::MaxRicciScalarCompute,
+        StrahlkorperTags::MinRicciScalarCompute,
+        StrahlkorperGr::Tags::ChristodoulouMassCompute<frame>,
+        StrahlkorperGr::Tags::DimensionlessSpinMagnitudeCompute<frame>>;
```

Comment:
Also add to EvolveGhBinaryBlackHole.hpp? I assume we'll need this quantity in the future.